content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+```shell
+maprcli s3keys generate -domainname primary -accountname default -username mapr -json
+```
+
+🗒Note: An Object Store cluster has a domain, accounts, buckets, users, and access policies associated with it.
+Installing Object Store in a cluster provides a primary domain and a default account.
+
+Sample output:
+
+```json
+{
+  "timestamp":1676472096994,
+  "timeofday":"2023-02-15 10:41:36.994 GMT+0800 PM",
+  "status":"OK",
+  "total":1,
+  "data":[
+          {
+            "accesskey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+            "secretkey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+          }
+  ]
+}
+```
+
+🗒Note: If you encounter any problem when generating the S3 keys, refer to this page: [Generate S3 Keys to Authenticate Users and Applications](https://docs.datafabric.hpe.com/72/MapROverview/object-store-get-started.html#object-store-get-started__section_x4h_w13_4tb).
+
+##### 3. Use the `mc alias set` command to create an alias for admin user
+
+```shell
+mc alias set s3-admin-alias https://`hostname -f`:9000 {ACCESS_KEY} {SECRET_KEY} --api "s3v4" --path "off" --json
+```
+
+🗒Note: "s3-admin-alias" is the name of the alias, you define it.
+
+"https://`hostname -f`:9000" is the endpoint of the Object Store service.
+Here, I'm running the command on the node which is running the S3server.
+After created an alias, you would find the information is appended into <ins>$HOME/.mc/config.json</ins>.
+
+##### 4. Create an account
+
+I choose to use the Object Store Web GUI to create an account.
+
+Refer to this document - [Using the Object Store Interface](https://docs.datafabric.hpe.com/72/MapROverview/create-account.html#create_account__section_fsm_1hn_nrb).
+
+Create an account using Object Store Web GUI - 1
+<a href="https://ibb.co/SyW9MKK"><img src="https://i.ibb.co/XLNvKzz/Object-Store-Create-Account-1.png" alt="Object-Store-Create-Account-1" border="0"></a>
+
+Enter the following in "Default Bucket Place":
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+        "Sid": "GrantAdminPutPermissions",
+        "Effect": "Allow",
+        "Principal": "arn:primary:default:user:mapr",
+        "Action":"s3:PutObject",
+        "Resource":"arn:aws:s3:::${bucket}/*"
+      },
+      {
+        "Sid":"GrantAnonymousReadPermissions",
+        "Effect":"Allow",
+        "Principal": "*",
+        "Action":["s3:GetObject"],
+        "Resource":["arn:aws:s3:::${bucket}/*"]
+      }
+  ]
+}
+```
+
+HPE Ezmeral Data Fabric Object Store is an on-premises object storage service compatible with Minio.
+Some concepts such as Domain and Default Account do not exist in public cloud object storage services such as AWS S3.
+But the policy for Bucket and IAM User is compatible with the policy in public cloud object storage.
```

Comment:
Consider whether these should be "Bucket" or "bucket" based on my previous comments.